tests/contrib/mysql/test_mysql.py

```diff
@@ -44,7 +45,7 @@ def test_simple_query(self):
 
         span = spans[0]
         assert_is_measured(span)
-        assert span.service == "mysql"
+        assert span.service == schematize_service_name("mysql")
```

Comment:
This is done to handle the case where the user runs the test via riot with `--pass-env` and `DD_TRACE_SPAN_ATTRIBUTE_SCHEMA = v1`.

I am open to removing or expanding this pattern.

Reply:
I think we shouldn't worry about users passing in envars to configure tracer behavior besides obviously needing the configuration to send to the testagent correctly.

Reply:
Actually, maybe we should remove it for now and then we'll add it everywhere when it becomes default. Kind of confusing to have it be spotty.

Reply:
Ack, I'll remove it for now.